ci/docker_ci.sh

```diff
@@ -20,6 +20,7 @@ set -e
 # AZP_BRANCH=refs/tags/v1.77.3
 ##
 
+
```

Comment:
nit: unnecessary new empty line.

Reply:
i left it because i figured altho an unncessary change the ws was desirable